src/main/java/com/tylrd/DemoApplication.java

```diff
@@ -32,6 +32,7 @@ ProtobufHttpMessageConverter protobufHttpMessageConverter() {
 
     @PostMapping(value = "/", produces = "application/json")
     ResponseEntity<Person> createPerson(@RequestBody Person person) {
+      DemoRockset d = new DemoRockset();
```

Comment:
```suggestion
      DemoRockset.main();
```
This will run the code defined in the static method DemoRockset.

Tests will fail with 

```
java.lang.NoClassDefFoundError: com/rockset/client/RocksetClient
    at com.tylrd.DemoRockset.main(DemoRockset.java:31)
```
 with this change.